ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/network_services_key.schema.yml

```diff
@@ -0,0 +1,1000 @@
+                items:
+                  type: dict
+                  keys:
+                    name:
+                      type: str
+                      description: BGP peer group name
+                    remote_as:
+                      type: int
+                      convert_types:
+                        - str
+                      description: Remote ASN
+                    description:
+                      type: str
+                    send_community:
+                      type: str
+                      description: |
+                        'all' or a combination of 'standard', 'extended', 'large' and 'link-bandwidth (w/options)'
+                    next_hop_self:
+                      type: bool
+                    maximum_routes:
+                      type: int
+                      convert_types:
+                        - str
+                      min: 0
+                      max: 4294967294
+                    default_originate:
+                      type: dict
+                      keys:
+                        enabled:
+                          type: bool
+                        always:
+                          type: bool
+                    update_source:
+                      type: str
+                    ebgp_multihop:
+                      type: int
+                      convert_types:
+                        - str
+                      min: 1
+                      max: 255
+                    nodes:
+                      type: list
+                      items:
+                        type: str
+                      description: |
+                        Nodes is required to restrict configuration of BGP neighbors to certain nodes in the network.
+                        If not set the peer-group is created on the device which has a bgp_peer mapped to corresponding peer_group.
+                    set_ipv4_next_hop:
+                      type: str
+                      description: |
+                        IPv4_address
+                        Next hop settings can be either ipv4 or ipv6 for one neighbor, this will be applied by a uniquely generated route-map per neighbor.
+                        Next hop takes precedence over route_map_out.
+                    set_ipv6_next_hop:
+                      type: str
+                      description: |
+                        IPv6_address
+                        Next hop settings can be either ipv4 or ipv6 for one neighbor, this will be applied by a uniquely generated route-map per neighbor.
+                        Next hop takes precedence over route_map_out.
+                    route_map_out:
+                      type: str
+                      description: Route-map name
+                    route_map_in:
+                      type: str
+                      description: Route-map name
+                    local_as:
+                      type: int
```

Comment:
BGP ASN must _always_ be a string (with convert from int), since "65001.1200" is a valid ASN.